BrainPortal/app/controllers/data_providers_controller.rb

```diff
@@ -468,10 +469,12 @@ def register
     succeeded, failed = [], {}
 
     CBRAIN.spawn_with_active_records_if(
-      [:html, :js].include?(request.format.to_sym),
+      #[:html, :js].include?(request.format.to_sym),
+      false,
       current_user,
       "Register files DP=#{@provider.id}"
     ) do
+      binding.pry
```

Comment:
This makes the rails app block forever expecting a programmer to enter the console, which doens't exist in production